src/decorators/relationships.js

```diff
@@ -18,32 +18,15 @@ import { singularizeType } from '../utils'
  * @method relatedToMany
  */
 export function relatedToMany (targetOrModelKlass, property, descriptor) {
-  if (typeof targetOrModelKlass === 'function') {
```

Comment:
Do we know for sure that it's safe to remove this check?

Reply:
Yes I removed the one place in artemis store that used it already

Reply:
It was an unnecessary one and we decided to remove the functionality - we can always bring it back if we need it down the road
https://github.com/artemis-ag/artemis-js/pull/288